src/index.tsx

```
@@ -1,14 +1,25 @@
-import { createRoot } from 'react-dom/client';
-import { HashRouter as Router } from 'react-router-dom';
+// import { createRoot } from 'react-dom/client';
+import { HashRouter } from 'react-router-dom';
 
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
-
+import ReactDOM from 'react-dom';
 import { App } from './App';
 
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

You should not ignore errors, you should fix them